composer.json

```diff
@@ -19,7 +19,7 @@
   },
   "require": {
     "ezsystems/ezpublish-kernel": "^6.3",
-    "ezsystems/landing-page-fieldtype-bundle": ">=1.3 <=1.7"
+    "ezsystems/landing-page-fieldtype-bundle": ">=1.3 1.7.*"
```

Comment:
Given we only support 1.7, lets simplify this to just `~1.7.0`

Reply:
@kmadejski Is this correct or not so much?

Reply:
I think so, good catch @andrerom 👍